index.css

```diff
@@ -0,0 +1,142 @@
+/* colors
+--clr-navbar-bg:22 1% 24% 1;
+--clr-base-bg:22 6% 10% 1;
+--clr-heading:22 100% 48%;
+--clr-text:0, 0%, 100%, 1;
+*/
+
+body {
+  margin: 0;
+  padding: 0;
+  display: flex;
+  flex-direction: column;
+}
+
+#header {
+  width: 100%;
+}
+
+.nav {
+  background-color: hsla(22, 1%, 24%, 1);
+  width: 100%;
+  height: 72px;
+  display: flex;
+
+  /* flex-direction: column; */
+  flex-wrap: wrap;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.name {
+  margin-left: 25px;
+  font-family: 'Roboto', sans-serif;
+}
+
+.pic {
+  /* width: 6%;
+    height: 20%; */
+  order: 1;
+  margin-right: 18px;
+}
+
+h4 {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 20px;
+  line-height: 20px;
+  color: hsla(0, 0%, 100%, 1);
+}
+
+.intro {
+  background-color: hsla(22, 6%, 10%, 1);
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  padding: 10px 24px 180px 24px;
```

Comment:
Great start @shuja-shah.🤝
- [ ] According to the Figma design provided. There seems to be a big white space left at the bottom of your design. You may please fix this margin-bottom issue so that your UI is consistent with the Figma design.
- Your design. 
![Your-design](https://user-images.githubusercontent.com/63917725/176701954-6f3d2d11-2aab-4b72-a48f-6c56922de102.PNG)
- Figma Design
![Figma-design](https://user-images.githubusercontent.com/63917725/176702108-59a2bb75-e3a6-4050-ad2f-d1426ac82f1d.PNG)